src/components/Locations.js

```diff
@@ -13,7 +13,8 @@ export default function Locations() {
     "Austin, TX",
     "San Francisco, CA",
     "Burlington, VT",
-    "Oshkosh, WI"
+    "Oshkosh, WI",
+    "Melbourne, VIC, AUS"
```

Comment:
```suggestion
    "Oshkosh, WI"
```
Don't change this yet! We'll update it after organizer registrations close.